### PR TITLE
Frontend oscar

### PR DESCRIPTION
**Pull request by Oshulten**

### CODE REVIEW

**Review by timabloom — APPROVED** [review]

File: Frontend/src/api/schema.d.ts
Comment (rorychatt) [inline Frontend/src/api/schema.d.ts:1]: once again, i suggest that the auto-generated types file should be gitignored and perhaps even moved somewhere else

**Review by rorychatt — APPROVED** [review]
good job!

File: Frontend/src/api/schema.d.ts
Comment (Oshulten) [reply Frontend/src/api/schema.d.ts:1]: Good suggestion, we'll fix that before merging into main

Comment (Oshulten) [reply Frontend/src/api/schema.d.ts:1]: Done

**Pull request merged**